samples/java-customer-registry/src/main/java/customer/CustomerValueEntityHandler.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package customer;
+
+import com.akkaserverless.javasdk.Effect;
+import com.akkaserverless.javasdk.Reply;
+import com.akkaserverless.javasdk.impl.*;
+import com.akkaserverless.javasdk.impl.valueentity.AdaptedCommandContext;
+import com.akkaserverless.javasdk.lowlevel.ValueEntityHandler;
+import com.akkaserverless.javasdk.valueentity.CommandContext;
+import com.google.protobuf.Any;
+import com.google.protobuf.Empty;
+import com.google.protobuf.GeneratedMessageV3;
+import customer.api.CustomerApi;
+import customer.domain.CustomerDomain;
+import scala.None$;
+import scala.Option;
+
+import java.lang.reflect.InvocationTargetException;
+
+/** This class would be generated by codegen */
+public class CustomerValueEntityHandler implements ValueEntityHandler {
+  final CustomerValueEntity entity;
+  final AnySupport anySupport;
+
+  CustomerValueEntityHandler(CustomerValueEntity entity, AnySupport anySupport) {
+    this.entity = entity;
+    this.anySupport = anySupport;
+  }
+
+  @Override
+  public Reply<Any> handleCommand(Any command, CommandContext<Any> context) throws Throwable {
+    AdaptedCommandContext<CustomerDomain.CustomerState> adaptedContext =
+        new AdaptedCommandContext(context, anySupport);
+    try {
+      // TODO we used to support passing in the command as Jackson-parsed model object
+      // or as ScalaPB class as well. With this change we tie ourselves to Java protobuf.
+      Effect<? extends GeneratedMessageV3> effect = invoke(command, adaptedContext);
+      // TODO we used to support accepting ScalaPB and Jackson objects as responses as well.
+      //  Are we OK with losing that? I guess we could have separate methods for that on the
+      //  builder, though it's not obvious how to achieve type-safety for those then.
```

Comment:
yes, I think we still want the json support. Could perhaps be an additional proto option on the entity (or even rpc) and we would generate different conversion.

Reply:
Isn't the json support some specific proto type that we could identify here (and turn to some json model) or how did that actually work?

(Somewhat confusing when there is also transcoding from json to proto in the proxy)